lessons/pydata/databases/index.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>Jak si to poradí s typama sloupců? Můžeš je tu vypsat?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>